library/core/src/main/java/com/google/android/exoplayer2/ExoPlayerImplInternal.java

```diff
@@ -257,7 +261,8 @@ public ExoPlayerImplInternal(
 
     Handler eventHandler = new Handler(applicationLooper);
     queue = new MediaPeriodQueue(analyticsCollector, eventHandler);
-    mediaSourceList = new MediaSourceList(/* listener= */ this, analyticsCollector, eventHandler);
+
+    mediaSourceList = new MediaSourceList(/* listener= */ this, analyticsCollector, prftListener, eventHandler);
```

Comment:
There is call to "updateLivePlaybackSpeedControl" inside "handleMediaSourceListInfoRefreshed()"
I wonder if we should invoke the prftListener there, then we don't need interface change on MediaSourceList

Reply:
`updateLivePlaybackSpeedControl()` has nothing to do with `prftListener`
`prftListener` is to listen for MP4 PRFT box updates, and it will be added in `MediaSourceList.mediaSourceEventDispatcher`